infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -409,6 +424,7 @@ EOT
     }
     "qualtrics" = {
```

Comment:
I didn't notice this before: is this source kind OK? `Integration.json` seed doesn't define `sourceKindId` for this connector

Reply:
it uses the generic importer (`survey-import-psoxy`), with parser=SURVEY or something like that. Calling it `qualtrics` just distinguishes it, and makes it clearer what kind of survey data the customer is dealing with. 

https://github.com/Worklytics/psoxy/pull/236/files/872e3e5980ff562fed1002af56a791706533909a..fb1c9e17eaea0e165b87452cd5ab366f9a66d8b2#diff-a95861f4521ed65205892ca0024ea417b0cdf21bd828a3eb3126811b7556bdeeR426-R429

all the connectorIds are explicitly defined there.

the instance id in the proxy isn't used for anything but distinguishing the different lambdas on the AWS side, really.  In theory, customer could have multiple instances mapping to same connector on Worklytics-side (different connections), with same source Id. eg (`gcal-1`, `gcal-2`, etc); our examples are organized for the usual case of single connection per so, but nothing about the proxy modules themselves constrains this.